src/decorator-observable.js

```diff
@@ -9,6 +9,11 @@ export function observable(targetOrConfig: any, key: string, descriptor?: Proper
 
     // use a convention to compute the inner property name
     let innerPropertyName = `_${key}`;
+    const innerPropertyDescriptor: PropertyDescriptor = {
+      configurable: true,
+      enumerable: false,
+      writable: true,
+    };
```

Comment:
Is this still useful with the new implementation of the setter? What happens if you remove it?